include/mbgl/util/feature.hpp

```diff
@@ -11,11 +11,35 @@ using Value = mapbox::base::Value;
 using NullValue = mapbox::base::NullValue;
 using PropertyMap = mapbox::base::ValueObject;
 using FeatureIdentifier = mapbox::feature::identifier;
-using Feature = mapbox::feature::feature<double>;
+using FeatureGeoJSON = mapbox::feature::feature<double>;
 using FeatureState = mapbox::base::ValueObject;
 using FeatureStates = std::unordered_map<std::string, FeatureState>;       // <featureID, FeatureState>
 using LayerFeatureStates = std::unordered_map<std::string, FeatureStates>; // <sourceLayer, FeatureStates>
 
+struct feature_mbgl : FeatureGeoJSON {
```

Comment:
`class Feature` - it's already inside `mbgl` namespace

Reply:
`: public FeatureGeoJSON` ?

Reply:
Done